l10n_br_cte/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+
+    <record id="cte_version_name" model="ir.config_parameter" forcecreate="True">
+        <field name="key">l10n_br_cte.version.name</field>
+        <field name="value">Odoo Brasil OCA v14</field>
+      </record>
```

Comment:
aqui poderia ter um campo generico para usar tanto pra cte quanto nfe ou qualquer outro doc

Reply:
blz.. mas se for o caso tratamos depois ok ?